en/getting_started/px4_basic_concepts.md

```diff
@@ -1,36 +1,81 @@
+A drone is an unmanned "robotic" vehicle that can be remotely or autonomously controlled.
 
-If you are already familiar with the basic concepts, you can move on to [Basic Assembly](https://docs.px4.io/en/assembly/) section that explains how to wire your specific autopilot hardware. To load firmware and set up the vehicle with QGroundControl, visit the section [Basic Configuration](https://docs.px4.io/en/config/).
+Drones are used for many consumer, industrial and military [use cases and applications](http://px4.io/applications/). These include (non exhaustively): aerial photography/video, carrying cargo, racing, search and surveying etc. 
+
+> **Tip** Different types of drones exist for use in air, ground, sea, and underwater. These are (more formally) referred to as Unmanned Aerial Vehicles (UAV), Unmanned Aerial Systems (UAS), Unmanned Ground Vehicles (UGV), Unmanned Surface Vehicles (USV), Unmanned Underwater Vehicles (UUV).
+
+The "brain" of the drone is called an autopilot. It consists of *flight stack* software running on *vehicle controller* ("flight controller") hardware.
+
+
+## Dronecode Platform {#dronecode}
+
+PX4 is part of the [Dronecode Platform](https://www.dronecode.org/platform/) — a complete end-to-end platform for drone development, delivered under a common industry-friendly open source license. It includes, among other things, the [PX4 flight stack](#autopilot)), [QGroundControl](#qgc) ground control station, the [Dronecode SDK](https://www.dronecode.org/sdk/) and the [Dronecode Camera Manager](https://camera-manager.dronecode.org/en/).
+
+## PX4 Autopilot {#autopilot}
+
+[PX4](http://px4.io/) is powerful open source autopilot *flight stack*. 
+
+Some of PX4's key features are:
+- Controls [many different vehicle frames/types](../airframes/airframe_reference.md), including: aircraft (multicopters, fixed wing aircraft and VTOLs), ground vehicles and underwater vehicles. 
+- Great choice of hardware for [vehicle controller](#vehicle_controller), sensors and other peripherals.
+- Flexible and powerful [flight modes](#flight_modes) and [safety features](#safety).
+
+
+## QGroundControl {#qgc}
+
+The Dronecode ground control station is called [QGroundControl](http://qgroundcontrol.com/).  You can use *QGroundControl* to load (flash) PX4 onto the [vehicle control hardware](flight_controller_selection.md), you can setup the vehicle, change different parameters, get real-time flight information and create and execute fully autonomous missions.
+
+*QGroundControl* runs on Windows, MacOS or Linux. Download and install it from [here](http://qgroundcontrol.com/downloads/). 
```

Comment:
Is Android missing here on purpose?

Reply:
No. That is what it said previously and I missed. Fixing now.